src/main/java/club/sk1er/patcher/mixins/features/GuiPlayerTabOverlayMixin_ReadablePing.java

```diff
@@ -46,7 +46,7 @@ public class GuiPlayerTabOverlayMixin_ReadablePing {
 
             GlStateManager.pushMatrix();
             GlStateManager.scale(0.5f, 0.5f, 0.5f);
-            mc.fontRendererObj.drawStringWithShadow("   " + (ping == 0 ? "?" : ping), (2 * x) - 10, 2 * y, color);
+            mc.fontRendererObj.drawStringWithShadow("   " + (ping == 0 ? "?" : ping == 9999 ? " " : ping), (2 * x) - 10, 2 * y, color);
```

Comment:
This 9999 value isn't used anywhere in Minecraft, a server is probably returning this to intentionally hide ping.

Reply:
I'll probably drop this then and come back if I find a better solution.